ui/dashboards/src/views/ViewDashboard/ViewDashboard.tsx

```diff
@@ -45,11 +48,29 @@ export function ViewDashboard(props: ViewDashboardProps) {
     sx,
     ...others
   } = props;
-  const { spec } = dashboardResource;
+  const { metadata, spec } = dashboardResource;
   const dashboardDuration = spec.duration ?? DEFAULT_DASHBOARD_DURATION;
-  const dashhboardRefreshInterval = spec.refreshInterval ?? DEFAULT_REFRESH_INTERVAL;
+  const dashboardRefreshInterval = spec.refreshInterval ?? DEFAULT_REFRESH_INTERVAL;
   const initialTimeRange = useInitialTimeRange(dashboardDuration);
-  const initialRefreshInterval = useInitialRefreshInterval(dashhboardRefreshInterval);
+  const initialRefreshInterval = useInitialRefreshInterval(dashboardRefreshInterval);
+
+  // Collect the Project variables and setup external variables from it
+  // TODO: Once we'll implement global variables CRUD, we'll do the same with 'global' as source
+  const { data: projectVars, isLoading: isLoadingProjectVars } = useVariableList(metadata.project);
```

Comment:
Checking twice how the components are architectured, it seems that the external variable retrieval should be done a bit upper and passed as a property